src/DDoSX/DomainClient.php

```diff
@@ -67,6 +68,32 @@ public function create(Domain $domain)
             });
     }
 
+    /**
```

Comment:
Because this is a POST action, wouldn't it fit our standards more to supply the method with a built Domain entity, e.g. the following:

```php
    /**
     * @param Domain $domain
     * @return |null
     */
    public function deploy($domain)
    {
        $this->post('v1/domains/' . $domain->name . '/deploy');

        return null;
    }
```

Reply:
I've updated the method based on your suggestion